build/js/tests.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+// This file is just a file that helped me in testing password and mail without having to manually write the same stuff over and over again
+// and yeah - I need to learn some tool for unit tests
+
+var primitiveTest = function primitiveTest(tested, against) {
```

Comment:
Here "var primitiveTest = " is useless. The named function primitiveTest is already available as a local variable.